README.md

```diff
@@ -860,6 +861,10 @@ COVERAGE=yes bundle exec rake parallel_spec
 Acceptance tests for this module leverage [puppet_litmus](https://github.com/puppetlabs/puppet_litmus).
 To run the acceptance tests follow the instructions [here](https://puppetlabs.github.io/litmus/Running-acceptance-tests.html). You can also find a tutorial and walkthrough of using Litmus and the PDK on [YouTube](https://www.youtube.com/watch?v=FYfR7ZEGHoE).
 
+## License
+
+This codebase is licensed under the Apache2.0 licensing, however due to the nature of the codebase the open source dependencies may also use a combination of [AGPL](https://www.gnu.org/licenses/agpl-3.0.en.html), [BSD-2](https://opensource.org/license/bsd-2-claus), [BSD-3](https://opensource.org/license/bsd-3-claus), [GPL2.0](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html), [LGPL](https://opensource.org/license/lgpl-3-0/), [MIT](https://opensource.org/license/mit/) and [MPL](https://opensource.org/license/mpl-2-0/) Licensing.
```

Comment:
2 things:
* Does it make sense to use a single website to access "details" of each license?  AGPL-v3 is available on opensource.com: https://opensource.org/license/agpl-v3/, some for GPL-2.0: https://opensource.org/license/gpl-2-0/
* The BSD license URL seems to have been trimmed. But they seems to redirect to the correct locations: https://opensource.org/license/bsd-2-clause/ / https://opensource.org/license/bsd-3-clause/. Maybe better use the final location?

I saw a bunch of other PRs, this apply to these too.